process.go

```diff
@@ -21,7 +17,4 @@ type ProcessConfig struct {
 	Stdin  io.Reader
 	Stdout io.Writer
 	Stderr io.Writer
-
-	// Console is the path to the pty slave for use by the master
-	Console string
```

Comment:
I think this is a valid point. Also, we could have more process specific parameters like capabilities, user, rlimits, etc. Should we have duplicate fields of what is in container config here and these take precedence? Or go back to not flattening it so it can be completely overriden per process?